gammapy/estimators/ts_map.py

```diff
@@ -146,20 +132,12 @@ def __init__(
         model=None,
         kernel_width="0.2 deg",
         downsampling_factor=None,
-        method="root brentq",
-        error_method="covar",
         n_sigma=1,
         ul_method="covar",
```

Comment:
Probably we should remove the `ul_method` choice as well? I think computing the upper limits on the covariance errors is not really a good idea, so probably we should just rely on likelihood profiles...